code/modules/mob/living/carbon/xenomorph/powers.dm

```diff
@@ -341,7 +341,7 @@ GLOBAL_LIST_INIT(acid_spray_hit, typecacheof(list(/obj/structure/barricade, /obj
 		return
 
 	to_chat(src, "<span class='notice'>You start focusing your plasma towards [target].</span>")
-	if(!do_after(src, transfer_delay, TRUE, 5, BUSY_ICON_FRIENDLY))
+	if(!do_after(src, transfer_delay, TRUE, null, USER_ICON_FRIENDLY) || QDELETED(target))
```

Comment:
Why not put the target in the place of the null?

Reply:
Because it would require the target to stay still instead of simply remain in range, that's how the ability currently works..